Hymns/Display/DisplayHymnView.swift

```diff
@@ -3,47 +3,35 @@ import Resolver
 
 struct DisplayHymnView: View {
 
+    @Environment(\.presentationMode) var presentationMode
     @ObservedObject private var viewModel: DisplayHymnViewModel
 
+    init(viewModel: DisplayHymnViewModel) {
+        self.viewModel = viewModel
+    }
+
     var body: some View {
         VStack {
-            GeometryReader { geometry in
-                VStack {
-                    HStack {
-                        NavigationLink(destination: HomeContainerView()) {
-                            Image(systemName: "xmark").accentColor(Color.black)
-                        }
-                        Spacer()
-                        if self.viewModel.favorited == true {
-                        }
-                        //TODO: Favorited needs to actually do something when clicked other than just say "favorited"
-                        Spacer()
-                        Button(action: {self.viewModel.toggleFavorited()}) {
```

Comment:
Any reason in particular you nixed my heart button from working? I can put it back in later but just wanted to check

Reply:
ah oops. Might have been an oversight on my part. Could you add it back later, when you add the favorites with realm?